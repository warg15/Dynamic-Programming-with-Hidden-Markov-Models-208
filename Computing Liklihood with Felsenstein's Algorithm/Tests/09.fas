>Great-cormorant
ATTTGGTACAGCTTAGAAATAAGATATTGGCAGGACCCAATGTGAGGATGACTATCATAGCTCGTATTGTTTATGGGAGAAACGGATCCAAAGCATGACGTGCCAAACGGTACGGGCGTATGGGCTACACAAGGGATGAATATAAATGGGTTGAATGACGAAAGAACGCTTTTAAATTTGTAGAGTCCGAATACATCATG
>Yellow-throated-sandgrouse
ATTTCGTACAGCTTAGAAATGAGATATTGGCAGGACGTAATGTGAGGTTGATTATCATAGCTCGTATTGTATATGGGCGAAACGGATCCAAAGCATTATGTGCCAAACGGAACGGGCATATAAACTACACAAGGGATAAATATAATAGGTTTAAATGACGAAGGAACGCTTTTATATTTGCTGAGTCCAAATATTTCATG
>Carmine-bee-eater
ATTTGGTATAGCTTGGATATGAGATATTGGCGGGACCTAATGTGAGGATGACCATCATGGCTCAGATCGTTTGTGGGAGAAACAGATCCAAAGCATGATGTGCCAAATGGTGAGGGCGTATGAACAACAAAAGGGAGGAATATAAATGGGTTAAATGACATGGGAACGCTCTTAGATCTCCAGAGGCCGTATACTTCATG
>Rhinoceros-hornbill
ATTTGGTATGGCCTAGGAATGAGGTCTTGGCAGGACCTAATGTGAGGATGATTCTCATAGCTCGTATTGTCTATAGGAAAAACAGATGCAAAGCATTACGGGCCAAACGATACGGGCACATGGACTACACTAGGGATGAATATAAATGGGTTAGATGGTAAGGGAACGGTTTTGCATTTGCAGCGTTCGAATACTTCATG
>Annas-hummingbird
AATTGGTACAGCTTAGTAATGTGATAATGGCAGTACCTAATGTGAGAATGATCCTCACACCTCGTATTGTTTATGAGAGGAACGGATCTAAAGCATGATGTACCCAAAGGTACGGGCGCATGAAATACACAAGGGATGAATATAAGTGGGTGAAACGGCGAAGGAACGTTTTTATAGTTGCAGAGTCCCAATACTTCATG
>Downy-woodpecker
ACTCGGTATAGCGTAGAAATGAGACATTGACAGGACCTAATGTAACGATGACTATCGTGGTTCGTATTGCTTATGGGAGAAACGGATCCACAGCATGATGTTCTAAACGGTACGGGCATATGGGTAACACTCGAGATGACTATAAATGAGTTAAATGTGGAAAGAACGTTTTTATATTTGCAGCGGCCGAATATCTCATG
>Golden-collared-manakin
ATTTGGTACAGCTTAGAAATGAGATATGGATAGGATCTAATGTGAGGATGGTTGTCATAGCTCATATTGATTGTGGGAGAAACGCCCACAAAGCATGACGTGACAAACGGTATGGGCATATGAACTACATAAGGGATGAATATAGATAGGTTAAATGGAGAAGAAACACTTTTACATTTGCAGAGTTCGAATGGTTCATT
>Sunbittern
ATTTGGTCCAGCTTAGAAATGTGATATTGGCAGGACCTTATGTGAGGATGATTATCATAGCTCGTATTATTTATGGGAGAAACGGATCCAAGCCATTATGTGCCAAACGATACGGGCATTTGAACTGAACAAGGAATGAATACAAATGGGTTAAGTGGCGGAGGAACGCTTCTATATTTGCAGGGTCCGAGTACATCATG
>Red-throated-loon
ATTTGGTATAGCTTAGAAATGAGATATTGGCAGGACCTAATGTGAGGATGATTATTATAGCTCTTATTGTTTATGGGAGAAACGCATCCAAAGCATGATGTGCCAAACGGTACGGGCATATGAACTACACAAGGGATGAGTATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGAAGAGTCCGACTACTTCATG
>Cuckoo-roller
ATTTGGTACAGCTTAGAAATGAGATATTGGCAGGACCTAATGTGAGGGTCATTATCGTAGCTCGTATTGTTTGTGGGAGAAACGGATCCAATGCGTGATGTGCCAAACGGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACTTCATG
>Red-crested-turaco
ATCTGGTCCAGCTTAAAAATGAGACATTGGCAGGACCGAATATGAGGATGATTATCATAGCTCGTATTGTTTATGGGAGACACTAGTCCAAAGCATGATGTGCCAAACGGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTCATGTTTGCAGAGTCCGAATACTTCATG
>Northern-fulmar
ATTTGGTACAGGTTAGAAATGAGATATTGGCAGGACCTAATGTGAGGATGATTATCATAGCTCGTATTGTTTATGGGAGAAACGGATCCAAAGCATGATGTGACAAACGGTACGGGCATATGAGCTACACAAGGGATGAATAGAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACTTCATG
>Pigeon
ATTTGCCACAGCTTAGAATTGGGATATTGGCAGGACCTAATGTGAAGAGGATCATCATAGCACGTATTGTTTATGGGAGGAATGGATACAAAGCATTATGTGCCAAACGGTACGGGCATATAAACTGCACAAGCGATGAATATCAATGGGTTAGATGACGAGGGAATGCTTTTATATTTGCAGAGTTCGAATACTCCATG
>American-flamingo
ATTTGGTACAGCTTAGAAATGAGATCTTGGCAGGAACTAATGTGAGCATGGTTATCATAGCTCGTATTGTTTATGGGAGAAACGGATCCAAAACATTATGTGCCAAACGGTACGGGCAAATAAACTACACAAGGGACGAATGTAAATGAGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGGGTCCGAATACTTCATG
>Turkey
AGTTGACAGAGTTTAGAAAGGAGATATTGTAATAGTCTGAAGTGAGGATGACAATTATAGGTTGCATTGTTTATTGGAGTAAGGGATCTAAAACAGTATCTACTGAACAGTACGGGCATATCAACCACGCACGGGATGGATATAAGCGGGTTAAATGGCTGGGGAACGCTTGTATAACTGGAAAGCCCGAATATTTCATG
>Chimney-swift
ATTTGGTACAGCTTAGAAATGCGATAATGGCATGACCTAATATAAGGATGATCTTCACACCTCGTATCGTCTATGGGAGGAACGGATCCAAAGCATGATCTGCTAAACGCTACGGGCATATGAACTACACAAGGAATGAAAATTAATGGGTTAAATGGCGAAGGAACGCTTTTACATTTACAGAGTCAGAACACTTCATG
>Dalmatian-pelican
ATTTGGTACAGCTTAGAAATGAGATATTGGCAGGACCTAATGTGAGGATGATTATCATAGCTCGTATTGTTTATGGGAGAAACGGATCCAGAGCATGATGTGCCAAACGGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACTTCATG
>MacQueens-bustard
ATTTGGTTCGGATTAGAAATGAACTATTGCCAGGACCCAATGTGAGGATGACTATCTTAGATCGTATTGTCTCTGGGAGAAACGGATCCAAAGCACGATGTGGTAAACGGTAAGGTCACATGAACTACACAAGGGATGAATATGAATGGGTTAAATGGCGAACGAACACTTTTATATTTGCAGAGTCCGTATACTCCATG
>Medium-ground-finch
ATTTGGTACACCTTAGAAATAAGATACCGACAGGAATTAATATGAGGATGGTTATCATAGCTCATATTGTTTATTGGAGAAACGGTTAGAAAGCATGATGTGACAAACGGTGCGCACATATGAACAACACATGGGATGAATATAGATGGATTAAATGGCGAAGGAACGCTTTTACATTCGCAGAATTCGAATGCTTCATG
>White-tailed-tropicbird
ATTTGGTATAGCTTAGAAATGTGATATTGGCAGGACCTAATGTGAGGATGATTATCATAGCTCGTATTATTTATGGGAGAAACGGATCAAAAGCATAATGTGCCAGACGGTACGGAAATATGAAGTACACAAGGAATGGATATAAATGAGTTAAATGGCGAGGGTACGCTTTTAGGTTTGCAGAGTCCGAATACTCCATG
>Bar-tailed-trogon
ATTCGGTACATTTTAGAAATGAGATAGCGATAGGCCCTAATGTGAGGATGATTATCATAGCTTGTATTGTTTATGGGAGAAACGGATCCAAAGGATAATGTGCTGATCGGTACGCGCATATGAACAACACAAGGGATGAATATAAGTGGGTTAAATGACGAAGGATCGCTTTTATATTTGCAGCGTCCGCATATTTCATG
>Rifleman
ATTTGGTACAGCTTAAAAATGAAATATCGGCAGGATCTAAGGTGAGGATGATTATCATAGCTCATATTGTTGATAGGAGAAACGGCTACAAACCATGATGTGCCAAACGGTACGGGCATATGAAACACACAAGGGGTGAACACAGCTGGGTTGAACGGCGAAGGAACGCTTCCACATTTGCAGAGTTCGACCCCTTCATG
>Brown-mesite
ATTTGGTACAGCTTAGAAATAAGATATTGGCATGACCTAATGTGAGGATGATTATCGTAGCTCGTATTCTTTATGGGAGAGACGGATCCAAATCATTATGTGCCAAACGGAACGGGCAAATAAACTACACAAGGGGTGAATATTAGTCGGTTAAATGGCGAGGGAACGCTTTTATATTTGCAGAATCCGAATACTCCATG
>Adelie-penguin
ATTTGGTACAGCTTAGAAACGAAATATTGGCAGGACCTAATGTGAGGATGATTAACATAGCTCGTATTGTTTATGGGAGAAACGGATCCAAAGCATGATGCGCCAAACAGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACCTCATG
>Chicken
ATTTGACAGAGTTTAGAAAGGAGATATTGGAAGAAGCTGATGTGAGGATGACAATTATAGATTGCGTTGTTTATTGGAGAAACGGATCTAAAACAGTATCTGCTGAACAATACGGGCATATCAACCACGCAAGGGATGGATATGAACGGCTTAAATGGCTGGGGAACGCTTATATAACTGTAAAGCCCGAATAATTCATG
>Common-ostrich
ATTTGGTATAGGTTAGAAATGAGATATTAGTAGGACTTGGAGGTAGACTGATTATTATCGATCGTATTGTTTATGGGATAAACGAATCTGAAACATTGTGTGCCAAAGGGTACGGGCATATAAGCTACGTAAGGGGTGAATATGAATGGGTTAAATAGCTGGGTAACGCTTTCATATTTCCGAAGTCCGGATACTTCATG
>Killdeer
ATTTGGTACAGCTTAGAAATGCGATATTGGCAGGACCTAATGTGAGGATGATTATCATAGCTCGTATTGTTTATGGGAGAAACGGATCCAAAGCATGATGTGCCAAACGGTACAGGCATAGGAACTACACAAGGGATGAATAAAAATGGGTTAAATGGCGAAGGAACGCTTCTATACCTGCAGAGTCCGAATACTTCATG
>Kea
ATTTGGTGCAGCTTAGTAATAAGATATTGGCAAGACTTAATGTGAGGTTGATTATCATAGCTCATATTGTTTACGGGAGAAACGGATACAGAACATGGTGTGCCAATCAGTACGGGCATATGAACTACACAAGGGATGAAGATAAATGGGTTAAACGGCGAAGGAACGTTTTTATATTTGCAGAGTCCGAATACTTCATG
>Zebra-finch
ATTTGGTACACCTTAGAAATAAGATACAGACAGGAATTAAGATGAGCATGGTTATCATAGCTCACGTTGTTTATTGGAGAAACGGTTACAAAGCATGATGTGACAAACGGTACGCACATATGAACCACACAAGGGATCACTATAGATGGGTTAAATGGCGAAGGAACGCTTTTACATTTGCAGAATTCGAATGCTTCATG
>American-crow
ATTTGGTACACCTTAGAAATGAGATACCGACAGGAATTAATATGAGGATGATTATCATAGCTCATATTGTTTATTGGAGAAACGGTTACAAAGCTTGATGGGACAAATGGTATGAACATATGAGCCACACAAGGGATGAATATAGATGGGTTAAATGGCGAAGGGACGCTTTTACATTTGCAGAATTCGAAAGCTTCATA
>Budgerigar
ATGTGGTGCAGCTTAGTAATGAGATATTGACAGGACTTAATGTGAGGATGATTATCATAGCTCATACTGTTTATGAGAGAACCGGATACAATGCATGATGTGCCAAACAGTACGGGCGTATGAACTACGCAGGGGATGAAATTAGATGGATTAAATGGCGAAGGAACGCTGATATATTTGCAGAGTCCAAATATTTCATG
>Hoatzin
CTTTGGTACACCTTAGAAGTGAGATATTGGCGGGACCTAATGTGAGGATGATTATCATAGTTCGTATTGTTTATGGGAGAAACAGATCCAAAGCATGATGTGCTAAACAGTACGGGCATATGAACAACATAAGGGATGAATATAAAAGGGTTAAAAGGCAGAGGAATGCTTTTATATTTGCAGAGTCCGAATACTTGATG
>White-tailed-eagle
ATTTGGTACAGCTTGTAAATGAGATATTGGCAGGACCTAATGTGAGGACGATTATCATAGCTCGTATTGTTTATGAGAGAAACGGATCCAAAGCAGGATGTGTCAAACGGTACGGGCATATGGACTACACAAGGGATGAATATAAATGGGTTAGATGGCGAAGGAACGCTTTTAAATTTGCAGAGTCCAAATACTTCATG
>Chuck-wills-widow
ATTTGGTACAGCTTAGAAATAAGATATTGGCAGGATCTAATGTGAGGAGGATTATCATAACTCGTATTGTTTATGGGATTAACGGGTCGAAAGCATGATTTGCCAAACGGTACGGGCCTATGAACTACACAAGGGACGAATATAAATGGGTTAAATGGCGAAGGAATGCTTTTTTATTTCCAGAATTCGAATATTTCATA
>Pekin-duck
ATTTGATACAGCTTAGAAATGAGAAATTAGCAGGACCTAATGTGAGGATGATTATTATGGATTGTATTTTTTATGGGAGAAGGAGACCTACAACAGCTTGTGCCCAATAGTACGGGCATAGCAACCACGCAAGGGATGGATATGAACGGGTTAAATAGCTAGGGAACACTTTTATATTTGTGAAGTCCGAATACTTCATG
>Speckled-mousebird
ATATGGTACAGGTTAGAAACTAGATATTGGCAGGGACTAATGTGAGGGTAATTATCATAGCTCCTATTATTTATGGGAGAAACGGACCCAAAGCACGATGTGCCAAACGATATGCACATATGAACTAAGCAGGGGATGAGTATAAATGTGTTAAATGGCGAAGGAACGCTTTTATGCGCGCAGGGTCCGAATACTTCGTG
>Emperor-penguin
ATTTGGTACAGCTTAGAAACGAGATATTGGCAGGACCTAATGTGAGGATGATTAACATAGCTCGTATTGTTTATGGGAGAAACGGATCCAAAGCATGATGCGCCAAACGGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATAGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACTTCATG
>Peregrine-falcon
ATTTGGTACAGCCTAGAAATGAGATATTGGCAGGACCTAATGTGAGGATAATTATCATAGCTTGTATTGTTTACGGAAGAAACGGATACAGAGCATGATGTACCAAACGGTACAGGCATATGAACCAAACAAGGGGTGAATACAAATACGTTAAATGGCGAAGCGACGCGTTTATATTTGCAGAGTCCGAATACTTCATG
>Common-cuckoo
ATTTGGCACAGCTTAGCAATGAGATATTCGTAGGACCTAATGTGAGGATGATCATCATAACTCATATTTTGTATGGGAGAAACGGATTCAAAGCGCGATGTGCCAAACGGTACGGGTATATGAACTATTCAAGGGATGAATTTAAATGGGTTAGACGGCGAAGGGACGCTTTTATATTTGCAGAGTCCGAATACTTTATG
>Red-legged-seriema
ATTTGGTACAGCTTAGAAATGAGATATTGGCAGGGCCTAATGTGAGGATGATTATCATAGCTCGTATCGTTTATGGGAGAAACGGATACAAAGCATGATGTGCCAAACGGTAAGGGCATATGAACTACACAAGACACGAATATAAATGGGTTAAATCGCGTAGGAACTCTTTTATATTTGCAGATTCCAAATACTTCATG
>Little-egret
ATTTGGTACAGTTTAGAAATGAGATATTCGCAGGACCTAATGTGAGGATCATTATCATAGCTTGTATTGTTTATGGGAGAAACGGATCCAAAGCATGATGTGCTAAACGGTACAGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATCTGCAGAGTCCGAACACTTCATG
>Grey-crowned-crane
ATTTGGTACAGCTTAGAAATGAGATATTGGCAGGACCTAATGTGAGGATGGTTATTATAGCTCGTATTGTTTATGGGAGAATCAGATCCAAAGCATGATGCGCCAAACGGTACGAGCATATGAATTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACTTTATG
>Crested-ibis
ATTTGGTACAGCTTAGAAATGAGATAGTGGCATGACCTAATGTGAGGATCATTATCATAATTCGTATTGTTTACGGGAGAAACGGATCTAAAGCATGATGTGCCAAACGGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTCATCTTTGCAGAGTCCGAATACTTCATG
>Bald-eagle
ATTTGGTACAGCTTGTAAATGAGATATTGGCAGGACCTAATGTGAGGACGATTATCATAGCTCGTATTGTTTATGAGAGAAACGGATCCAAAGCAGGATGTGTCAAACGGTACGGGCATATGGACTACACAAGGGATGAATATAAATGGGTTAGATGGCGAAGGAACGCTTTTAAATTTGCAGAGTCCAAATACTTCATG
>Turkey-vulture
ATCCGGTACAGCTTAGAAATGAAATATTGGCAGGACCTAATGTGAGGATGATTATCATAGCTCGTATTATTTATGAGAGAAACGGATCCAAAGCATGATGTGCGAAACGGTACGGGCATATGAACTACACAAGGGATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTTGCAGAGTCCGAATACTTCATG
>White-throated-tinamou
ATTTGGTATAAGTTAGAAATGAGATATTGGTAGAACTTAGAGGGAGAATCATTATTCCCGATCGTATTGTTTATGGGACAAACGAATCTGGAATAGTGTGTACCATGCGGTGCGAGCATAGGAACTAAGTAAGGGGTGAATATGAATGGGTTAGATAACGAAGTAAGTGTTTCATATTTCCGATATCCAAATACTTCATT
>Great-crested-grebe
ATTTGCTACAGCTTAGAAATGAGATATTGACAGGACCTAAAGTGAGGATGATCATCATAGCTCGTATTGTTTATGGGAGGAACGGATCCAAAGCATTATGTGGCAAATGGTACGGGCATATATTCTACACAAGAAATGAATATAAATGGGTTAAATGGCGAAGGAACGCTTTTATATTGGCAGAGTCCGAATACTTTATG
>Barn-owl
ATTTGATACAGCTTAGAAATGAGATATTGGCAGGACCTAATGTGAGGCTGATTATCATAGCTTGTATTGTTTATGGGAGAAATGGATCCAAAGCATGATGTGCCAAACGGTACGGGCATATGAACTACACAGAGGATGAACATAAATGGGTTAAATGGCGAAGGAACGCTATTATATTTGCAGAGTCCGAATACTTCATG
