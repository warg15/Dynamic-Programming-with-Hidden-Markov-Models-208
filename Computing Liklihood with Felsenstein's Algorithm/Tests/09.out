-3619.0903181720555
